config.microsoftInternal.dsc

```diff
@@ -11,7 +11,7 @@ export const pkgs = isMicrosoftInternal ? [
     { id: "BuildXL.DeviceMap", version: "0.0.1" },
 
     // Runtime dependencies used for macOS deployments
-    { id: "runtime.osx-x64.BuildXL", version: "1.99.99" },
+    { id: "runtime.osx-x64.BuildXL", version: "2.0.1" },

```

Comment:
Did kext version remain the same in this NuGet?

Reply:
Yes, only interop went up - kext is still signed